app/features/text_rewriter/requirements.txt

```diff
@@ -0,0 +1,38 @@
+fastapi
+fastapi[standard]
+uvicorn[standard]
+langchain
+langchain-core
+langchain-google-genai
+langchain-chroma
+langchain-community
+langgraph
+google-cloud-secret-manager
+google-cloud-logging
+google-auth
+google-cloud-storage
+firebase-admin
+chroma
+pypdf
+fpdf
+youtube-transcript-api
+pytube
+unstructured
+python-pptx
+docx2txt
+networkx
+pandas
+xlrd
+openpyxl
+gdown
+pytest
+PyPDF2
+python-dotenv
+psutil
+pydub
+ffmpeg-python
+speechrecognition
+google-cloud-speech
+python-docx
+google-cloud-aiplatform
+vertexai  # No need for vertexai.generative_models, it's already included in vertexai
```

Comment:
Why is this dependency needed?

Reply:
I used this because I had trouble using langchain and could only get the model to work via google gemini and a vertex ai API key. With the examples you provided I'm rewriting the code to get rid of the vertext ai API.